README.md

```diff
@@ -1,4 +1,4 @@
-# shipator
+<img src="./logo.svg" height="120" alt="Shipator" />
```

Comment:
@diogomoretti  The readme is also deployed to docker hub. And in there we can't use relative references as `./logo.svg`. 
Would be possible to get a absolute link to github.